web/client/src/entities/Ddi/Form.tsx

```diff
@@ -10,12 +11,39 @@ const Form = (props: EntityFormProps): JSX.Element => {
     const { entityService, row, match } = props;
     let properties = props.properties;
 
+    const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+
+    const skip: Array<string> = [];
+    if (!aboutMe?.pbx) {
+        skip.push(...[
+            'user',
+            'ivr',
+            'huntGroup',
+            'conditionalRoute',
+            'conferenceRoom',
+            'queue',
+        ]);
```

Comment:
Again, there are other vpbx endpoints like friend, conditionalRoute.
Fax again is present in residential and vpbx afaik

Reply:
Again